NewChallenge1.js

```diff
@@ -0,0 +1,36 @@
+
+//CHALLENGE 1
+
+// Make a program where you input 2 names using prompt (2 separate prompts after one another) and it prints (
+//console.log) what percentage chance there is that they would get married/be friends/work well together/whatever you think is fun.
+// - So you'd create 2 prompts that would store the names in 2 different variables (like the quiz answer)
+// - Make a function that outputs a random percentage (0-100%)
+// - And then console.log something like "George and Mendelsohn have a 92% chance of getting married"
+
+
+const nameOne = prompt("Enter name one")
+
+const nameTwo = prompt("Enter name two")
+
+const randomPercentage = () => {
+    return (
+        Math.floor(Math.random()*100)
+    )
+}
```

Comment:
When you aren't adding any extra logic in your function (above the return) you can make this an IMPLICIT return, which would look this this:

```const randomPercentage = () => (
        Math.floor(Math.random()*100)
)
```

Which is the same as what you have above. What you have allows you to add extra stuff like 
```const randomPercentage = () => {
        const newThing = 'test'
        console.log("Extra logic here")
        return (
               Math.floor(Math.random()*100)     
       )
}
```

Nothing to change, just pointing out the difference